src/Microsoft.Sbom.DotNetTool/FormatValidationService.cs

```diff
@@ -26,11 +29,26 @@ public class FormatValidationService : IHostedService
         this.hostApplicationLifetime = hostApplicationLifetime;
     }
 
+    private void PrintLines(List<string> lines)
+    {
+        foreach (var line in lines)
+        {
+            Console.WriteLine(line);
+        }
+    }
+
     public async Task StartAsync(CancellationToken cancellationToken)
     {
         try
         {
-            Console.WriteLine($"Format validation service called for {config.SbomPath.Value}");
+            using (var sbomStream = new StreamReader(config.SbomPath.Value))
+            {
+                var validatedSbom = new ValidatedSBOM(sbomStream.BaseStream);
+                var details = await validatedSbom.GetValidationResults();
+                var sbom = await validatedSbom.GetRawSPDXDocument();
```

Comment:
nit: we never appear to use the raw SBOM, why is this line necessary?

Reply:
It's not, nor the preceding. Good catch!

Reply:
Oh hey, that exposed a nice usage bug! The ValidatedSBOM is lazy-loaded but MultilineSummary was not loading it, so removing these 2 lines caused the validate-format command to stop outputting results to the CLI.